### DISCUSSION TITLE
Can I force the app to open to where I left it?

### DISCUSSION THREAD

**Discussion by geoffmozz** [osmandapp/OsmAnd#18293] [General]
Is there a settings change I can make in OsmAnd, so that when I open the app I'm looking at the map where I left it?

I'm looking at routes in Nepal but live in England. It's a drag (literally) to find Nepal every time I open the app. I just want to open the app to the map of Nepal exactly where I left it, when I last closed the app.

**Comment by DmitryAlexei** [comment]
You can disable location services in Android before launching OsmAnd, so the center of the map will not be located where your location is. And the map will be opened where you left it.

**Reply by geoffmozz** [reply]
Thanks Dmitry. I was hoping for an OsmAnd setting but your suggestion will help occasionally, thanks.
